tock/tock/static/css/style.css

```diff
@@ -204,10 +204,10 @@ picture {
   max-width: 100%; }
 
 .banner {
```

Comment:
this css needs some formatting / linting cleanup in general -- i will do this when i get a chance soon -- but i don't think it should block this PR